l10n_it_account_balance_eu/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Associazione Odoo Italia (<http://www.odoo-italia.org>)
+# Copyright 2022 MKT Srl (<www.mkt.it>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# last update: 2023-05-23
+
+{
+    "name": "ITA - Bilancio UE con XBRL",
+    "version": "14.0.1.0.6",
```

Comment:
La versione di un nuovo modulo di norma è `14.0.1.0.0`, forse se ne è anche già parlato in questa PR e non sto trovando il commento.
Non credo comunque sia una modifica necessaria per arrivare al merge

Reply:
fatto